eng/Packages.Data.props

```diff
@@ -177,6 +176,11 @@
     <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="6.35.0" />
   </ItemGroup>
 
+  <!-- Azure.Identity is only allowed for non-Azure.* client libraries and test projects -->
+  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and !$(MSBuildProjectName.StartsWith('Azure.'))">
+    <PackageReference Update="Azure.Identity" Version="1.13.1" />
```

Comment:
We should move this under the '$(IsExtensionClientLibrary)' branch.

Reply:
This is a previous commit that you have since reverted. You need to reapply this to change in your branch.